Readme update

normancroan opened the pull request:
Added culling CPU image to Readme

Commit:
Before and after culling

Image depicting performance before and after utilizing SKATK culling
method

Commit:
added culling-performance image

Updated the Culling Feature section to include an image depicting the performance gains post-culling

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Horizontal comparison

Commit:
renamed original file and moved h

Commit:
Update README.md

pasta12 commented:
The image link will need to be a relative URL. `![Culling Performance](Documentation/culling-performance-h.png)` should theoretically work.

Commit:
Update README.md

Commit:
CollisionDefine for Swift

Added CollisionDefine.m and updated CollisionDefine.h to support
bridging into Swift.

normancroan commented:
Thanks, updated the Readme to include that relative URL.

pasta12 commented:
Closing this. New PR with the same change will be coming from another branch.
